pipeline.go

```diff
@@ -418,6 +418,7 @@ func (p Pipeline) KillContainers() error {
 	for _, pipeline := range *pipelines {
 		for _, step := range pipeline {
 			NewContainerKiller(step)()
+			NewOldContainerRemover(step)()
```

Comment:
Since there is no `NewContainerRemover` I'd propose calling it that to be less verbose and it's also not clear what makes a container old. Also this would make it consistent with the `NewNetworkRemover` function.